src/test/java/APITest/dataprovide.java

```diff
@@ -0,0 +1,35 @@
+package APITest;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import io.restassured.RestAssured;
+
+public class dataprovide {
+	
+	
+	@Test(dataProvider="Registerdetails")
+	public void userregistration(String name,String job,int id)
+	    {
+
+	   RestAssured.baseURI="https://reqres.in";
+	   String response= given().header("Content-Type","application/json")
+	             .body(Payload.create_multiple_user(name, job, id))
+	             .when().post("/api/users")
+	             .then().log().all().assertThat().statusCode(201).extract().response().asString();
+             
+		 System.out.println(response);
```

Comment:
Remove prints. They are useful when debugging or when writing the test, but should be removed from the final code so that it is easier to see which tests failed when tests are run in bulk.